src/styles/index.scss

```diff
@@ -1,3 +1,101 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import './blocks/card';
+@import './blocks/stars';
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+
+  --main-color: #00acdc;
+  --col-number: 1;
+}
+
+header {
+  background-color: white;
+  z-index: 1;
+  position: sticky;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+
+    &-item {
+      display: inline-block;
+      text-transform: uppercase;
+      text-align: center;
+
+      &:not(:first-child) {
+        margin-left: 20px;
+      }
+    }
+  }
+
+  &__link {
+    align-content: center;
+    display: inline-block;
+    color: #060b35;
+    text-decoration: none;
+    font-size: 12px;
+    height: 60px;
+    transition-duration: 300ms;
+
+    &:hover {
+      color: var(--main-color);
+    }
+  }
+}
+
+a.is-active {
+  color: var(--main-color);
+  position: relative;
+
+  &::after {
+    position: absolute;
+    content: '';
+    border-radius: 10px;
+    background-color: var(--main-color);
+    height: 4px;
+    right: 0;
+    bottom: 0;
+    left: 0;
+  }
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-right: 20px;
+}
+
+.cards {
+  display: grid;
+  grid-template-columns: repeat(var(--col-number), minmax(200px, 200px));
```

Comment:
The use of `minmax(200px, 200px)` in `grid-template-columns` is redundant since both the minimum and maximum values are the same. You can simply use `200px` instead.